dysnix/geth/values.yaml

```diff
@@ -265,3 +265,42 @@ config:
   pprof:
     enabled: false
     port: 6061
+
+### S3 directory structure
+### Currenly tested only 1 baseUrl per 1 node deployment
+###
+###   baseUrl
+###   ├── ancient_url
+###   └── chaindata_url
+###
+###   cat ancient_url   ->  baseUrl/<your_ancient_destination>
+###   cat chaindata_url ->  baseUrl/<your_chaindata_destination>
+###
+
+s3config:
+  image:
+    repository: peakcom/s5cmd
+    tag: v2.2.2
+    pullPolicy: IfNotPresent
+  baseUrl: S3-BUCKET-NAME/{{ .Release.Name }}
+  endpointUrl: https://storage.googleapis.com
```

Comment:
It may be good to add some AWS endpoints in comments, as long as this variable is mandatory